code/script.js

```diff
@@ -1,19 +1,62 @@
-// Start here
-
-// Step 1 - Welcome and introduction
-// Your code goes here
 alert(
-  `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
+  "Welcome to Pizzeria Penguin. Ready to order? - Click 'OK' to begin.")
+
+//user name//
+const userName = prompt("Please enter your name:")
+console.log(userName)
```

Comment:
Please remove the the console.log() around the code. Generally they need to be deleted when the code has been tested and works as meant to. 🧹

Reply:
Well noted and changes made, did not actually not know this.